core/admin/EE_Admin_Page_Init.core.php

```diff
@@ -386,8 +417,8 @@ protected function _initialize_admin_page()
                 'AHEE__EE_Admin_Page___initialize_admin_page__before_initialization_' . $this->_menu_map->menu_slug
             );
             require_once($path_to_file);
-            $a = new ReflectionClass($admin_page);
-            $this->_loaded_page_object = $a->newInstance($this->_routing);
+            $this->_loaded_page_object = $this->loader->getShared($admin_page, [$this->_routing]);
+            $this->_loaded_page_object->initializePage();
```

Comment:
split this up because child classes would call parent constructor
but the constructor was running all kinds of logic prior to child classes being able to finish instantiation